sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/aspnethostedservice/tests/Azure.Messaging.EventHubs.Processor.Samples.HostedService.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
```

Comment:
This will fail outside the Azure SDK repository.   We'll need to do a conditional definition to define these if we're not in the repo.   Since we've got a targets file, that'd be the place.

Reply:
@jsquire I'm not fully understanding the ask here. Can you give me an example of what you're looking for? I've looked for other examples in sample projects but can't find anything.

Reply:
@jsquire Understood. I've added `Directory.Build.props` and updated `Directory.Build.targets`. I'm forced to only target net6 and net7 in `Directory.Build.props`, as it's not possible to build against net462 for the aspnet app (net462 get's passed down by `Directory.Build.Common.props` , but I have to override ride it).  If there's a way to work around this, please let me know.

Reply:
The only way to work around it would be to screen it out unless `IsSamplesProject` is set;  the `net462` target is part of our required test matrix in the repo.   I think the approach that you've got for just hiding the tests is the best we can do.